src/math/warpExposure.cc

```diff
@@ -657,7 +948,7 @@ int afwMath::warpCenteredImage(
     t += dt;
     std::cout <<srcImage.getWidth()<<"x"<<srcImage.getHeight()<<": "<< dt <<" "<< t <<std::endl;
 #else
-    int n = warpImage(destImage, srcImageCopy, affXYTransform, control, padValue);
+    int n = warpImage(destImage, srcImageCopy, affXYTransform, control, covImage, padValue);
```

Comment:
The naming has been mangled a bit because of the route taken by me to hack at the old code and because of my inexperience with templated code going in. The first variant of dowarpCovariance warps both the image and the covariance matrices while the second one warps just the image. If we end up turning any of this into production code, I'll clean the naming up (amongst a couple of other things that could be cleaned).